pkg/constants/env.go

```diff
@@ -31,11 +31,12 @@ const (
 	AnnotationDefaultAutoInstrumentationApacheHttpd = InstrumentationPrefix + "default-auto-instrumentation-apache-httpd-image"
 	AnnotationDefaultAutoInstrumentationNginx       = InstrumentationPrefix + "default-auto-instrumentation-nginx-image"
 
-	EnvPodName  = "OTEL_RESOURCE_ATTRIBUTES_POD_NAME"
-	EnvPodUID   = "OTEL_RESOURCE_ATTRIBUTES_POD_UID"
-	EnvPodIP    = "OTEL_POD_IP"
-	EnvNodeName = "OTEL_RESOURCE_ATTRIBUTES_NODE_NAME"
-	EnvNodeIP   = "OTEL_NODE_IP"
+	EnvPodName        = "OTEL_RESOURCE_ATTRIBUTES_POD_NAME"
+	EnvPodUID         = "OTEL_RESOURCE_ATTRIBUTES_POD_UID"
+	EnvPodIP          = "OTEL_POD_IP"
+	EnvNodeName       = "OTEL_RESOURCE_ATTRIBUTES_NODE_NAME"
+	EnvNodeIP         = "OTEL_NODE_IP"
+	ReservedNamespace = "resource.opentelemetry.io/"
```

Comment:
How about we rename this to `OtelAnnotationNamespace`? Just `ReservedNamespace` runs the risk of it being confused for a normal K8s Namespace.

Reply:
done